src/openforms/registrations/contrib/json_dump/plugin.py

```diff
@@ -114,7 +115,10 @@ def get_variables(self) -> list[FormVariable]:  # pragma: no cover
 
 
 def post_process(
-    values: JSONObject, schema: JSONObject, submission: Submission
+    values: JSONObject,
+    schema: JSONObject,
+    submission: Submission,
+    transform_to_list: list | None = None,
```

Comment:
```suggestion
    transform_to_list: list[str],
```
with the change to types above